Experiment_Launcher_code/ModuleConfiguration.py

```diff
@@ -1,5 +1,5 @@
-__USE_VIDEO_SIM = False
-__USE_VIDEO_STUB = False
-__USE_ARDUINO_SIM = False
+__USE_VIDEO_SIM = True
```

Comment:
I know that this is just being picky :-) But please change it back to the default (not using the simulated module). I think main should have the "production" code